### PR TITLE
Balances Report All Token Ids In Wallet

### PR DESCRIPTION
**Pull request by BrianCorbin**

### CODE REVIEW

File: full-service/src/service/balance.rs [hunk 1]
```diff
@@ -67,21 +71,27 @@ impl From<LedgerServiceError> for BalanceServiceError {
...
     pub pending: u128,
     pub spent: u128,
     pub secreted: u128,
     pub orphaned: u128,
-    pub unverified: u128,
-    pub network_block_height: u64,
-    pub local_block_height: u64,
-    pub synced_blocks: u64,
-    pub max_spendable: u128,
+    // pub network_block_height: u64,
```
Comment (Shramp) [inline full-service/src/service/balance.rs:91]: looks like it's safe to delete these comments now

File: full-service/src/service/balance.rs [hunk 2]
```diff
@@ -266,21 +262,13 @@ where
...
-            conn,
-        )?
-        .max_spendable_in_wallet;
-
+    ) -> Result<Balance, BalanceServiceError> {
         let unspent = sum_query_result(Txo::list_unspent(
             account_id_hex,
             assigned_subaddress_b58,
-            Some(0),
+            Some(*token_id),
```
Comment (Shramp) [inline full-service/src/service/balance.rs:271]: what is the `*` doing here? according to SO it de-references the value? Not sure what that means or why it's being used here

Comment (Shramp) [inline full-service/src/service/balance.rs:271]: why the de-referencing with `*` here?

**Review by Shramp — COMMENTED** [review]
LGTM once commented code is deleted and tests pass! Stoked to get stablecoin into FS!

File: full-service/src/service/balance.rs (hunk 2 above)
Comment (BrianCorbin) [reply full-service/src/service/balance.rs:267]: it breaks the wrapper object TokenID and turns it into a u64, which is what the consuming function needs

**Pull request merged**